src/app/static/src/app/components/load/UploadNewProject.vue

```diff
@@ -5,7 +5,6 @@
             <input id="project-name-input"
                    type="text"
                    class="form-control"
-                   @keyup.enter="submitLoad"
```

Comment:
This was removed because I have not managed to synchronised it with valid entry. A user can just press enter and continue, even if project name is invalid. I have created a ticket to handle this here https://mrc-ide.myjetbrains.com/youtrack/issue/VIMC-6651